sdk/highlight-next/src/util/withHighlight.ts

```diff
@@ -108,6 +108,7 @@ export const Highlight =
 				// We rethrow here so that nextjs can do with the error whatever it would normally do. (Sometimes "whatever it
 				// would normally do" is to allow the error to bubble up to the global handlers - another reason we need to mark
 				// the error as already having been captured.)
+				await NodeH.stop()
```

Comment:
I was getting weirdly persistent processes in local dev. I suspect that our SDK's `setInterval` was keeping them alive indefinitely, effectively causing a memory/listener leak.